beetmoverscript/src/beetmoverscript/script.py

```diff
@@ -109,6 +109,26 @@ async def push_to_nightly(context):
     add_checksums_to_artifacts(context)
 
 
+# direct_push_to_releases {{{1
```

Comment:
We may want to rename this function to direct_push_to_bucket at some point since we're going to use it not only to push Fenix into ~releases direcrtly, but also in ~nightly buckets. Not blocking for now, but may help with confusion in the future.